types/jsreport-html-to-xlsx/index.d.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference types='jsreport-core' />
+/// <reference types='jsreport-xlsx' />
+
+declare namespace JsReport {
+	const enum RecipeType {
+		Html2Xlsx = 'html-to-xlsx'
+	}
+
+	interface Html2Xlsx {
+		shortid: string;
+	}
+
+	interface Template {
+		xlsxTemplate: Partial<Html2Xlsx>;
```

Comment:
The `html-to-xlsx` recipe adds just recipe type.  It doesn't extend the template. See the [source](https://github.com/jsreport/jsreport-html-to-xlsx/blob/master/lib/htmlToXlsx.js). You should keep here only the RecipeType changes.

Reply:
OK